### PR TITLE
Retrieve clinic

### PR DESCRIPTION
**Pull request by Elexy**
In this PR i will retrieve the Clinic files and store them on IPFS.

### CODE REVIEW

File: runner/runner.js [hunk 1]
```diff
@@ -26,15 +30,20 @@ const run = async (commit) => {
     // first run the benchmark straight up
     try {
       let result = await runCommand(test.benchmark, test.name)
+      config.log.debug(result)
+      fs.writeFileSync(`${targetDir}/${test.name}/results.json`, JSON.stringify(result))
```
Comment (mcollina) [inline runner/runner.js]: Can you please use a promise version of this? Use util.promisify.

File: runner/runner.js [hunk 2]
```diff
@@ -15,6 +17,8 @@ const runCommand = (command, name) => {
 }
 
 const run = async (commit) => {
+  const targetDir = `${os.tmpdir()}/${Date.now()}`
+  fs.mkdirSync(targetDir, { recursive: true })
```
Comment (mcollina) [inline runner/runner.js]: use await and a promise version.

File: runner/runner.js (hunk 1 above)
Comment (Elexy) [reply runner/runner.js]: done

File: runner/runner.js (hunk 2 above)
Comment (Elexy) [reply runner/runner.js]: done

**Pull request merged**